test/blob.js

```diff
@@ -2,6 +2,7 @@
 
 var assert = require('assert');
 var hash = require('../index');
+var validSha1 = /^[0-9a-f]{40}$/i;
```

Comment:
This variable was missing, but somehow is magically present in older versions of Node.js. Presumably this is some kind of scoping issue, I am not quite sure how it ever worked without.